File: backend/src/main/java/ca/metricalsky/winston/entity/view/ChannelVideoView.java
package ca.metricalsky.winston.entity.view;

import ca.metricalsky.winston.database.entity.channel.ChannelEntity;
import ca.metricalsky.winston.entity.VideoEntity;

public interface ChannelVideoView {

    ChannelEntity getChannel();

    VideoEntity getVideo();
}

File: database/src/main/kotlin/ca/metricalsky/winston/database/view/ChannelVideoView.kt
package ca.metricalsky.winston.database.view

import ca.metricalsky.winston.database.entity.channel.ChannelEntity
import ca.metricalsky.winston.database.entity.video.VideoEntity

interface ChannelVideoView {

    val channel: ChannelEntity
    val video: VideoEntity
}
